server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java

```diff
@@ -852,10 +852,14 @@ public boolean test(TabletMetadata tabletMetadata) {
                   final FileSystem fs = vol.getFileSystem();
                   for (ExternalCompactionId ecid : ecidsForTablet) {
                     final String fileSuffix = "_tmp_" + ecid.canonical();
-                    FileStatus[] files = fs.listStatus(new Path(volPath), (path) -> {
-                      return path.getName().endsWith(fileSuffix);
-                    });
-                    if (files.length > 0) {
+                    FileStatus[] files = null;
+                    try {
+                      files = fs.listStatus(new Path(volPath),
+                          (path) -> path.getName().endsWith(fileSuffix));
+                    } catch (FileNotFoundException e) {
+                      LOG.trace("Failed to list tablet dir {}", volPath, e);
```

Comment:
Could just `continue` here?

Reply:
You could, the result is going to be the same either way so if you want to use continue instead in the catch that's fine. There was already an if statement checking the size files.length > 0 so i just switched that to check for null.